tools/model_tools/README.md

```diff
@@ -29,12 +29,12 @@ future releases.
 
 ## Installation
 
-Model Downloader and other automation tools can be installed as a part of OpenVINO&trade; toolkit or from source.
+Model Downloader and other automation tools can be installed as a part of [openvino-dev](https://pypi.org/project/openvino-dev/) Python\* package or from source.
 Installation from source is as follows:
```

Comment:
for me, installation from source means installing from repo, not from pypi

Maybe better separate this section on installing from source (maybe useful for contributors) and installing with PyPI?

Reply:
then probably you need explain why you need to install package, now it is unclear. Besides that they also can be installed from source.
Also instruction mislead with extras installation. You suggest install framework deps with extras and via requirements in the same time, however they should be optional (at least pytorch, as mo does not depend from it)

Reply:
Updated installation section